setup.cfg

```diff
@@ -1,2 +1,42 @@
+[metadata]
+name = QtPy
+description = Provides an abstraction layer on top of the various Qt bindings (PyQt5, PyQt4 and PySide) and additional custom QWidgets.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/spyder-ide/qtpy
+author = Colin Duquesnoy, The Spyder Development Team
+author_email = goanpeca@gmail.com
+maintainer = Gonzalo Peña-Castellanos
+maintainer_email = goanpeca@gmail.com
+license = MIT
+license_file = LICENSE.txt
```

Comment:
`license_file` is deprecated and `license_files` should be used instead, and in any case `LICENSE.txt` will be included automatically, so this can just be removed.

```suggestion
```